chaco/tests/test_colormapper.py

```diff
@@ -130,36 +130,3 @@ def test_no_alpha(self):
             "red": [(0.0, 0.0, 0.0), (1.0, 1.0, 1.0)],
         }
         assert self.colormap._segmentdata == sd
-
-
-##     def test_no_interpolation(self):
-##         grayscale_colors = array([[0.0,0.0,0.0,1.0], [1.0, 1.0, 1.0, 1.0]])
-##         grayscale_bins = array([0.0, 1.0])
-##         grayscale_steps = array([1])
-
-##         colormap = LinearSegmentedColormap(
-##             grayscale_colors, grayscale_bins, grayscale_steps
-##         )
-
-##         a = array([0.0, 0.25, 0.75, 1.0])
-##         b = colormap.map_array(a)
-##         result = ravel(b[:,:1])
-##         expected = array([0.0, 0.0, 1.0, 1.0])
-
-##         close = allclose(result, expected, atol=0.02)
-##         self.assert_(close,
-##             "Map with no interpolation broken.  Expected %s.  Got %s" % (expected, result))
-
-##     def test_value_bands(self):
-
-##         grayscale_colors = array([[0.0,0.0,0.0,1.0], [1.0, 1.0, 1.0, 1.0]])
-##         grayscale_bins = array([0.0, 1.0])
-##         grayscale_steps = array([1])
-
-##         colormap = LinearSegmentedColormap(
-##             grayscale_colors, grayscale_bins, grayscale_steps
-##         )
-
-##         colormap._recalculate()
-
-##         print '**************', colormap._color_bands, colormap._value_bands
```

Comment:
I haven't looked at this test/relevant code closely yet, but just the word interpolation reminded me of https://github.com/enthought/enable/issues/20
I am going to see if this test passes with the latest enable

Reply:
ah never mind, a search of `LinearSegmentedColormap` in the codebase yields no results outside of this file... So these tests are definitely good to go

The test case should probably be renamed to either `ColormapTestCase` or `TestColormap`.

Reply:
renamed